filters/reflection/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```

Comment:
I think you can replace this with your name.

Reply:
Thanks. @bigtimebuddy @englercj @GoodBoyDigital  , Shall I ?

Reply:
Yes that’s fine with me

Reply:
@bigtimebuddy  , OK , I'll create a new PR for changing all filters created be me.